libr/anal/class.c

```diff
@@ -127,6 +127,25 @@ R_API bool r_anal_class_exists(RAnal *anal, const char *name) {
 	return r;
 }
 
+R_API RBufVector/*<const char *>*/ *r_anal_class_get_all(RAnal *anal) {
```

Comment:
this seems not very efficient, sdb can return a list and you can convert it to vector for faster nth access in cutter. in r2land i dont see the need to have such access need.